packages/admin/admin-stories/src/docs/components/DateAndTimePickers/stories/DateTimePicker.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import { Field, FieldContainer } from "@comet/admin";
+import { DateTimePicker, FinalFormDateTimePicker } from "@comet/admin-date-time";
+import { Grid } from "@mui/material";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { Form } from "react-final-form";
+
+storiesOf("stories/components/Date & Time Pickers/Date-Time Picker", module)
+    .add("Basic", () => {
```

Comment:
This example doesn't work for me. The time doesn't change.

https://user-images.githubusercontent.com/13380047/187164194-990563e1-dc5a-4953-a244-31d7e8f9af2a.mov

(Only the basic example doesn't work. The Final Form example works)

Reply:
Yes, that's a weird bug where the story doesn't re-render on state-change for the time for some reason. 
Since the stories are often used as copy-paste examples, I didn't want to add a workaround inside the story, that is only needed for storybook...

But I'll see if I can find a proper fix for this.

Reply:
@thomasdax98 I have created a workaround proposal in #769

Reply:
@thomasdax98 Correction: It was a bug from me, fixed in #770